.github/workflows/ci.yml

```diff
@@ -644,6 +644,24 @@ jobs:
       ( needs.build-info.outputs.run-kubernetes-tests == 'true' ||
       needs.build-info.outputs.needs-helm-tests == 'true')
 
+  tests-task-sdk:
+    name: "Task SDK tests"
+    uses: ./.github/workflows/task-sdk-tests.yml
+    needs: [build-info, wait-for-ci-images]
+    permissions:
+      contents: read
+      packages: read
+    secrets: inherit
+    with:
+      runs-on-as-json-default: ${{ needs.build-info.outputs.runs-on-as-json-default }}
+      image-tag: ${{ needs.build-info.outputs.image-tag }}
+      default-python-version: ${{ needs.build-info.outputs.default-python-version }}
+      python-versions: ${{ needs.build-info.outputs.python-versions }}
+      debug-resources: ${{ needs.build-info.outputs.debug-resources }}
+    if: >
```

Comment:
Actually - there is a bug: 

https://github.com/apache/airflow/actions/runs/11417702941

The workflow is invalid, tests are not running because of that and (very unfortunately) GitHub will show the status of PR as green - but with only 7 checks passed. You need to add "run-task-sdk-tests" input parameter here.

Reply:
This is a very bad design from GH -> because you can see that the workflow is invalid only after you click on the summary page (so when you merge PRs you also have to see if a number of workflows is > 10 :(

I raised it like 2 years ago to GH... Not fixed till today. Such PR should have "status=failed" but it does not.

Reply:
aah yeah, good catch, fixed it in https://github.com/apache/airflow/pull/43185/commits/f28c4d7530d008abfd06a2eb28c7a328ae314778